### DISCUSSION TITLE
Capture Name/Email on Campaign message

### DISCUSSION THREAD

**Discussion by iammattmartin** [chatwoot/chatwoot#8470] [Feature Requests]
**Is your enhancement request related to a problem? Please describe.**
Yes

**Describe the solution you'd like**
When a campaign message is sent, a new conversation is started but then does not capture name/email. It should do this before the chat starts so we know who we are talking to.

Otherwise, this is going to revert back to the old behaviour of getting pointless messages. 

I would say this should be done either before a message can be sent, or after a message is sent but before it is delivered to staff.

**Comment by pranavrajs** [comment]
I'm moving these to discussions as we will not be prioritizing these immediately.
